gyp/sources.txt

```diff
@@ -1,4 +1,6 @@
 <(src_loc)/spellcheck/platform/platform_spellcheck.h
+<(src_loc)/spellcheck/platform/linux/enchant_linux.h
+<(src_loc)/spellcheck/platform/linux/enchant_linux.cpp
```

Comment:
=(

Please look at [Telegram/SourceFiles/platform/linux](https://github.com/telegramdesktop/tdesktop/tree/dev/Telegram/SourceFiles/platform/linux) folder.
"Cross-platform" code files has `linux` as the postfix and "single-platform" code files has `linux` as the prefix.
We don't have the Enchant on other platforms.
I'm sorry, but it would be better to keep the consistency with the TDesktop.

Reply:
Ah, sorry, I misunderstood the original structure of the source tree. Now I've renamed the file as you requested.